CHANGELOG

```diff
@@ -0,0 +1,107 @@
+==== 2.1 beta 2017-12-22 ====
+After two years of hard work, we are pleased to announce Mixxx 2.1 beta. We
+have overhauled the effects system, redesigned the skins, added and improved
+lots of controller mappings, rewrote the audio file decoders twice, and of
+course fixed a bunch of bugs. Download it!
```

Comment:
Is an unconditional "Download it!" a good idea? It is till quite "untested". 
How about: "Download it and hunt for bugs", or something less negative.
"Download it and help making it mature for release" ...

Reply:
I think it is fine. We are already announcing it as a beta release.

Reply:
Ok

Reply:
I agree this is fine the way it is.  The "mixxx voice" is lighthearted and energetic, so if you wanted to add an equivocal statement you could do something like:

"Download it! (But don't blame us if you find a new bug at peak-time!)"

Remember that most of our users are non-technical DJs, not linux sysadmins :).